### DISCUSSION TITLE
How to combine two sequencing data of the same sample/library?

### DISCUSSION THREAD

**Discussion by shanshancode** [satijalab/seurat#6678] [Q&A] [answered]
Hello,
I sequenced twice of the scRNA library (10x) because the first batch I did not get enough reads per cell.

I was wondering what will be the best practice to combine the two sequencing data?

1. run cellranger with the combined fasta? if so, has anybody done this before and could you let me know how two set the parameters?

2. combine the raw counts sparse matrix/or combine the object?
I did merge the two objects and got :

Warning message:
In CheckDuplicateCellNames(object.list = objects) :
  Some cell names are duplicated across objects provided. Renaming to enforce unique cell names.

Anybody have other solution? Many thanks!

I am relatively new here, sorry if I have repeated the same issue tho I have done the search.
Best,
Shanshan

**Comment by samuel-marsh** [comment, marked as answer]
Hi,

This needs to be done at cell ranger level otherwise you will not be appropriately accounting for UMIs.  You just need to provide two paths to each flowcells fastq data at the cell ranger count stage.
https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/count

Best,
Sam

**Reply by shanshancode** [reply]
Hi Sam,
This is helpful!
Many thanks,
shanshan
